release-notes/major10/10.7.0/valtimo-backend-libraries.md

```diff
@@ -24,6 +24,11 @@ The following features were added:
   The types of files uploaded to the temporary file storage can now be restricted by whitelisting allowed mime-types.
   More information on this feature can be found [here](/using-valtimo/upload/temporary-file-storage.md#whitelisting-file-types-for-uploads)
 
+* **New service ObjectsService, for access to ObjectenAPI and ObjecttypenAPI**
```

Comment:
Can you make a new PR on the major 11 branch? And then move these release notes you added to the 11.0.0 release notes as well please!